hudi-hadoop-mr/src/test/java/org/apache/hudi/hadoop/realtime/TestHoodieRealtimeRecordReader.java

```diff
@@ -237,13 +231,13 @@ public void testUnMergedReader() throws Exception {
     // create a split with baseFile (parquet file written earlier) and new log file(s)
     String logFilePath = writer.getLogFile().getPath().toString();
     HoodieRealtimeFileSplit split = new HoodieRealtimeFileSplit(
-        new FileSplit(new Path(partitionDir + "/fileid0_1-0-1_" + instantTime + ".parquet"), 0, 1, jobConf),
-        basePath.toString(), Collections.singletonList(logFilePath), newCommitTime);
+        new FileSplit(new Path(partitionDir + "/fileid0_1-0-1_" + instantTime + ".parquet"), 0, 1, baseJobConf),
+        basePath.toUri().toString(), Collections.singletonList(logFilePath), newCommitTime);
```

Comment:
@vinothchandar Tests are passing now. the issue is isolated with this test case. do you concern about this change masking the issue with Azure environment?

Reply:
i could spend more time digging into Azure env around this problem with must having scheme, but given this is a blocker, shall we move on and keep a note about this?